src/zenml/cli/server.py

```diff
@@ -120,8 +120,23 @@ def up(
     """
     from zenml.zen_server.deploy.deployer import ServerDeployer
 
+    if connect:
+        logger.warning(
```

Comment:
maybe add it here? https://www.notion.so/zenml/Deprecations-903bcaacb5c04954b9f6133b3340a312